jupyter-extension/jupyterlab_pachyderm/mount_server_client.py

```diff
@@ -23,10 +23,12 @@ class MountServerClient(MountInterface):
     def __init__(
         self,
         mount_dir: str,
+        sock_dir: str,
     ):
         self.client = AsyncHTTPClient()
         self.mount_dir = mount_dir
-        self.address = f"http://localhost:{MOUNT_SERVER_PORT}"
+        self.sock_dir = sock_dir
+        self.address = f"http://tmp/pfs_fuse_server.sock"
```

Comment:
I'm surprised that the unix domain socket still has http:// on the front when using it.   But perhaps.

Reply:
will see once I am able to test

Reply:
The code is using tornado AsyncHTTPClient() -- so this might be a path forward:

https://github.com/tornadoweb/tornado/issues/2671